caps/capabilities_test.go

```diff
@@ -56,8 +58,10 @@ func (s *CapabilitySuite) TestAdd(c *C) {
 
 func (s *CapabilitySuite) TestAddClash(c *C) {
 	repo := NewRepository()
+	err := LoadBuiltInTypes(repo)
```

Comment:
Feels a bit strange to be running this on every test, even though the test is unrelated to it.